packages/main/src/DatePicker.js

```diff
@@ -503,6 +521,81 @@ class DatePicker extends UI5Element {
 				this._toggleAndFocusInput();
 			}
 		}
+
+		if (this.isOpen()) {
+			return;
+		}
+
+		if (isPageUpShiftCtrl(event)) {
+			event.preventDefault();
+			this._incrementValue(true, true, false, false, 1);
+		} else if (isPageUpShift(event)) {
+			event.preventDefault();
+			this._incrementValue(true, false, true, false, 1);
+		} else if (isPageUp(event)) {
+			event.preventDefault();
+			this._incrementValue(true, false, false, true, 1);
+		}
+
+		if (isPageDownShiftCtrl(event)) {
+			event.preventDefault();
+			this._incrementValue(false, true, false, false, 1);
+		} else if (isPageDownShift(event)) {
+			event.preventDefault();
+			this._incrementValue(false, false, true, false, 1);
+		} else if (isPageDown(event)) {
+			event.preventDefault();
+			this._incrementValue(false, false, false, true, 1);
+		}
+	}
+
+	_incrementValue(increment, years, months, days, step) {
```

Comment:
The name of the method should be something neutral, as its action depends on its arguments. Something like mutateValue/changeValue... You can add default values here as well.

Reply:
Done